editor/src/components/inspector/sections/layout-section/layout-system-subsection/split-chained-number-input.tsx

```diff
@@ -240,7 +240,15 @@ const actionsForSplitChainedEvent =
         jsxAttributeValue(
           normalizedValues.length === 1 && normalizedValues[0] != null
             ? printCSSNumber(normalizedValues[0], 'px')
-            : mapDropNulls((v) => v, normalizedValues)
+            : mapDropNulls((v) => {
+                if (v == null) {
+                  return null
+                }
+                return {
+                  ...v,
+                  unit: useShorthand && v.unit == null ? 'px' : v.unit,
```

Comment:
so if `useShorthand` false, then unit: false? isn't that a problem?

Reply:
wdym? if `useShorthand` is false the unit will be `null` 🤔